dbsc/workflow/rules/mrtpipelines.smk

```diff
@@ -140,6 +140,7 @@ rule dwi2response:
 
 rule responsemean:
     """Compute average response function"""
+    # DOUBLE CHECK
```

Comment:
What are we double checking here?

Reply:
Ahh, this was a note to my self, I removed it in one of the other PRs. Just wanted to check the conditional was written correctly by comparing with an existing rule written in another workflow.